RebuildClient/Assets/Scripts/MapEditor/Editor/RagnarokWalkableDataImporter.cs

```diff
@@ -213,7 +213,7 @@ public RoMapData LoadWalkData(string path, float waterHeight)
                             color = "purple";
                             break;
                         default:
-                            throw new Exception("Unknown cell type " + type);
+                            throw new Exception("Unknown cell type: " + map.name + "(X: " + x + " / Y: " + y + ") " + type + ", Likely result of an incompatible map version. Stopping map import process.");
```

Comment:
I would suggest to keep the message short, and include only the necessary info so we know what happened, and for which resource.

```suggestion
                            throw new Exception("Unknown cell type for Map " + map.name + " (X: " + x + " / Y: " + y + ") Value " + type");
```